client/src/css/Aaron.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+h1,
+p,
+Table {
+  font-family: 'Inter', sans-serif;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h1 {
+  padding-top: 15vh;
+  font-size: 60;
+}
+
+body {
+  background-color: #daad86;
+}
+.animated {
+  animation: shapeIn 1s;
+}
+@keyframes shapeIn {
+  from {
+    opacity: 0;
+    transform: translateY(-20px);
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+hr {
+  border: 1px solid black;
+  width: 30%;
+  margin-left: 3vh;
+  margin-bottom: -1vh;
+}
+
+.main-heading {
+  font-family: 'Quicksand', sans-serif;
+  font-weight: 600;
+  font-size: 300%;
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 4vh;
+  margin-top: -11vh;
+}
+
+.sub-heading {
+  font-family: 'Quicksand', sans-serif;
+  display: flex;
+  justify-content: flex-start;
+  font-weight: 400;
+  margin-top: 1vh;
+  font-size: 120%;
+  margin-left: 3vh;
+}
+
+.bottomBar {
+  margin-top: -2px;
+}
+
+.navBar {
+  font-family: 'Quicksand', sans-serif;
+  display: flex;
+  flex-direction: row;
+  font-size: 90%;
+  justify-content: flex-start;
+  word-spacing: 15px;
+  margin-left: 3vh;
+  margin-top: -3vh;
+}
+
+.links {
+  transition: all 0.3s ease;
+}
+
+.intenseThought {
+  width: 20%;
+  display: flex;
+  float: right;
+  margin-right: 15vh;
+  margin-bottom: 4vh;
+  box-shadow: 1px 1px 20px black;
+  transition: all 0.3s ease;
+}
+
+.bruh {
+  border: 2px dashed black;
+  width: 45%;
+  display: flex;
+  justify-content: center;
+  margin-bottom: -10px;
+}
+
+.innards {
+  font-family: 'Quicksand', sans-serif;
+  display: flex;
+  text-align: left;
+  font-size: 120%;
+  padding: 3vh;
+  margin-top: -1vh;
+}
+
+.links:hover {
+  text-shadow: 2px 2px 10px black;
+}
+
+.categories {
+  display: flex;
+  justify-content: center;
+  font-family: 'Quicksand', sans-serif;
+}
+
+.innardsTwo {
+  display: flex;
+  justify-content: center;
+  font-size: 120%;
+  font-family: 'Quicksand', sans-serif;
+}
+
+p {
+  padding-top: 2em;
+}
+
+img {
+  border-radius: 50%;
+}
```

Comment:
remove these, since you're using class names for your `p` and `img` tags instead